core/controllers/base.py

```diff
@@ -148,7 +148,7 @@ class BaseHandler(webapp2.RequestHandler):
 
     URL_PATH_ARGS_SCHEMAS = None
     HANDLER_ARGS_SCHEMAS = None
-    ARGS_WHICH_DO_NOT_NEED_SCHEMA_VALIDATION = ['csrf_token']
+    ARGS_WHICH_DO_NOT_NEED_SCHEMA_VALIDATION = ['csrf_token', 'source']
```

Comment:
Please update [similar](https://github.com/oppia/oppia/pull/13225) changes to this PR also.